rust/agents/relayer/src/msg/metadata/multisig/merkle_root_multisig.rs

```diff
@@ -51,9 +55,15 @@ impl MultisigIsmMetadataBuilder for MerkleRootMultisigMetadataBuilder {
             return Ok(None);
         };
 
+        let merkle_leaf_id = self
```

Comment:
to be safe I think we should do the None check here and return early if this is None

I think it's unlikely we'd get None here if the above `get_proof` gives Some, but if we ever did, then we would end up hitting the `.ok_or(RelayerError::MetadataFetchingError)?` which will return an Err when we try to format the metadata. Which we really don't want to happen